src/Server/AspNetCore.Tests/Helpers/ClientQueryRequest.cs

```diff
@@ -8,5 +10,9 @@ internal class ClientQueryRequest
         public string NamedQuery { get; set; }
         public string Query { get; set; }
         public JObject Variables { get; set; }
+
+        [JsonIgnore]
```

Comment:
Since we are moving away from newtonsoft we should not introduce new attributes here.